addons/account/static/src/components/duplicate_bill_warninng/duplicate_bill_warning.js

```diff
@@ -0,0 +1,67 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import { Component } from "@odoo/owl";
+import { getCurrency } from "@web/core/currency"
+
+
+class DuplicateBillWarning extends Component {
+    static template = "account.DuplicateBillWarning";
+    static props = {};
+    setup() {
+        this.action = useService("action");
+        this.orm = useService("orm");
+    }
+
+    async discard() {
+        if (this.record?.isInEdition) {
+            await this.record.discard();
+        }
+    }
+
+    async open_bills_tree_and_discard() {
+        await this.discard();
+        const action = await this.orm.call("account.move", "open_duplicated_ref_bill_view", [this.props.record.resId], {});
+        this.action.doAction(action);
+    }
+
+
+    async open_bill_form_and_discard(id) {
+        await this.discard();
+        this.action.doAction({
+            type: "ir.actions.act_window",
+            res_model: "account.move",
```

Comment:
I'm wondering if this should be not a bit more generic?

You already know `res_model` from the field definition

Reply:
sure !